modules/Bio/EnsEMBL/Compara/PipeConfig/BuildNewMasterDatabase_conf.pm

```diff
@@ -0,0 +1,229 @@
+        # Parameters required for 'test' division only
+        'config_dir'    => undef,
+        'reg_conf_tmpl' => undef,
+        'dst_host'      => undef,
+        'dst_port'      => undef,
+
+        # PrepareMasterDatabaseForRelease pipeline configuration:
+        'taxonomy_db'             => 'ncbi_taxonomy',
+        'incl_components'         => 1, # let's default this to 1 - will have no real effect if there are no component genomes (e.g. in vertebrates)
+        'create_all_mlss_exe'     => $self->check_exe_in_ensembl('ensembl-compara/scripts/pipeline/create_all_mlss.pl'),
+        'xml_file'                => $self->o('config_dir') . '/compara_' . $self->o('division') . '.xml',
+        'report_file'             => $self->o('work_dir') . '/mlss_ids_' . $self->o('division') . '.list',
+        'master_backup_file'      => $self->o('backups_dir') . '/new_master_' . $self->o('division') . '.sql',
+        'patch_dir'               => $self->check_dir_in_ensembl('ensembl-compara/sql/'),
+        'alias_file'              => $self->check_file_in_ensembl('ensembl-compara/scripts/taxonomy/ensembl_aliases.sql'),
+        'list_genomes_script'     => undef, # required but not used: do_update_from_metadata = 0
+        'report_genomes_script'   => undef, # required but not used: do_update_from_metadata = 0
+        'update_metadata_script'  => $self->check_exe_in_ensembl('ensembl-compara/scripts/pipeline/update_master_db.pl'),
+        'assembly_patch_species'  => [], # by default, skip this step
+        'additional_species'      => {}, # by default, skip this step
+        'do_update_from_metadata' => 0,
+        'do_load_lrg_dnafrags'    => 0,
+        'do_load_timetree'        => 0,
+    };
+}
+
+sub pipeline_create_commands {
+    my ($self) = @_;
+    return [
+        # Inherit creation of database, hive tables and compara tables
+        @{$self->SUPER::pipeline_create_commands},
+
+        $self->pipeline_create_commands_rm_mkdir(['work_dir', 'backups_dir']),
+    ];
+}
+
+sub pipeline_wide_parameters {
+    # These parameter values are visible to all analyses, can be overridden by parameters{} and input_id{}
+    my ($self) = @_;
+    return {
+        # Inherit anything from the base class
+        %{$self->SUPER::pipeline_wide_parameters},
+
+        'master_db'     => $self->o('master_db'),
+        'division'      => $self->o('division'),
+        'release'       => $self->o('ensembl_release'),
+        'hc_version'    => 1,
+        # Define the flags so they can be seen by Parts::PrepareMasterDatabaseForRelease
+        'do_update_from_metadata' => $self->o('do_update_from_metadata'),
+        'do_load_lrg_dnafrags'    => $self->o('do_load_lrg_dnafrags'),
+        'do_load_timetree'        => $self->o('do_load_timetree'),
+    };
+}
+
+sub pipeline_analyses {
+    my ($self) = @_;
+    return [
+        {   -logic_name => 'create_new_master',
+            -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
```

Comment:
This analysis and the ones below feel like they should be executed using `Bio::EnsEMBL::Hive::RunnableDB::DbCmd` but the latter doesn't support a custom registry. You can leave this as it is for now

Reply:
Yes, that is why I chose this option.